charts/spring-boot/spring-boot-chaos/engine.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: spring-boot-app-chaos
+  namespace: default
+spec:
+  appinfo:
+    appns: 'default'
+    applabel: 'app=springboot'
+    appkind: 'deployment'
+  # It can be active/stop
+  engineState: 'active'
+  chaosServiceAccount: spring-boot-chaos-sa
+  experiments:
+    - name: spring-boot-delete
+      spec:
+        components:
+          env:
+            - name: TOTAL_CHAOS_DURATION
+              value: '15'
+
+            ## Period to wait before injection of chaos in sec
+            - name: RAMP_TIME
+              value: ''
+
+            ## env var that describes the library used to execute the chaos
+            ## default: litmus. Supported values: litmus
+            - name: LIB
+              value: 'litmus'
+
+            # Chaos Monkey Spring Boot configuration
+            # Level determines at which frequency the assault happens on the application. For a value N, the assaults happens every N requests
+            - name: CM_LEVEL
```

Comment:
Can we only add the mandatory variables here in the engine while the optional variables (which can be supplemented with a default value in the `experiments.yaml` file)

Reply:
ok

Reply:
IMO, these variables are mandatory to be provided for the chaos by the user so they can be added to engine.yaml
```
      # Comma separated list of watched Java services. Ex: com.example.application.controller.HelloController.sayHelle
      - name: CM_WATCHED_CUSTOM_SERVICES
        value: ''

      # Comma separated list of watchers. Possible values: controller, restController, service, repository, component, restTemplate, webClient, actuatorHealth
      - name: CM_WATCHERS
        value: 'restController'
```

These two can also be provided in engine.yaml, not sure if the user would require to change them frequently

```
# Level determines at which frequency the assault happens on the application. For a value N, the assaults happens every N requests
      - name: CM_LEVEL
        value: '1'

      # Whether the level should be used as a deterministic value (attack every x requests) or a chance (on average, 1 in x requests will be attacked)
      - name: CM_DETERMINISTIC
        value: 'true'
```
If none of these are, we can continue with the current engine.yaml config.